src/language-css/utils/index.js

```diff
@@ -54,6 +54,17 @@ function getPropOfDeclNode(path) {
   return declAncestorNode?.prop?.toLowerCase();
 }
 
+const wrappedProps = new Set([
+  "box-shadow",
+  "transition",
+  "background-image",
```

Comment:
Not only `background-image`, we should treat all longhand `background-*` the same. https://developer.mozilla.org/en-US/docs/Web/CSS/background#constituent_properties